DOI.js

```diff
@@ -93,52 +94,83 @@ function completeDOIs(doc) {
 	}
 }
 
-function retrieveDOIs(DOIs, doc) {
-	__num_DOIs = DOIs.length;
 
+var providers = [
+	"11645bd1-0420-45c1-badb-53fb41eeb753", //CrossRef
+	"9f1fb86b-92c8-4db7-b8ee-0b481d456428"  //DataCite
+];
+
+
+/*
+var providers = [
+	"9f1fb86b-92c8-4db7-b8ee-0b481d456428", //DataCite
+	"11645bd1-0420-45c1-badb-53fb41eeb753"  //CrossRef
+];
+*/
+
+function retrieveDOIs(DOIs, doc, provider) {
```

Comment:
While we're modifying this, let's steer away from using variable names that start with caps. `dois` instead of `DOIs`